### PR TITLE
Implementing table

### PR DESCRIPTION
**Pull request by carakessler**
Implementing table UI component and added tests. Screenshot attached

### CODE REVIEW

**Comment by carakessler** [comment]
Screen Shot 2020-04-10 at 1 48 42 PM

File: packages/cli/__tests__/report-components/table-test.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  meta: {
+    taskName: 'mock-task',
+    friendlyTaskName: 'Mock Task',
+    taskClassification: {
+      category: Category.Migrations,
+      priority: Priority.Medium,
+    },
+  },
+  result: [
+    { name: 'ember-cli ember-cli ember-cli ember-cli ember-cli ember-cli', value: '12.2.2' }, // testing long name
```
Comment (scalvert) [inline packages/cli/__tests__/report-components/table-test.ts:14]: I get the `ember-cli` part, but do you have to repeat yourself?

Comment (carakessler) [reply packages/cli/__tests__/report-components/table-test.ts:14]: i was using that to stress test the layout, i can remove it if you dont want it committed to the repo

Comment (scalvert) [reply packages/cli/__tests__/report-components/table-test.ts:14]: Ya it's really hard to grok what you're trying to accomplish here. If it's a reaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaly long name, then just use `reaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaly-long-name` maybe?

Comment (carakessler) [reply packages/cli/__tests__/report-components/table-test.ts:14]: i commented `// testing long name`! but ya ill swap out with `reaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaly-long-name`

**Pull request merged**